docs/Data Model.md

```diff
@@ -9,11 +9,18 @@ In order to ensure consistent behaviour in a facility employing this identity, s
 
 ![Extended IS-04 Model Diagram](../docs/images/is-11-data-model.png)
 
-The IS-04 heirarchy is preserved with the exception of the addition of a Receiver can identify an association with 0 or more Sink resources.
+The IS-04 heirarchy is preserved with the exception of the addition of Sender can identify an association with 0 or more Input resources and Receiver can identify an association with 0 or more Output resources.
 
-## Sink Representation
+## Input Representation
 
-A new Sink is created in the following circumstance:
+A new Input is created in the following circumstance:
+
+- At the presence of an IS-11 compatible Sender
```

Comment:
Not sure what this mean ... this first lie is different from the Receiver side ...

Reply:
```suggestion
- At the presence of a physical source or another producing unit
```
What about this?

Reply:
Yes, it was in my head at that moment. If we create Inputs without EDID for SDI sources, what is the use case for Sender without Inputs? The same can be asked about Receiver without Outputs.